js/widgets.js

```diff
@@ -1,108 +1,172 @@
-Array.range = function(n) {
-    // Array.range(5) --> [0,1,2,3,4]
-    return Array.apply(null, Array(n)).map((x, i) => i);
-};
+/* This file is using http://standardjs.com/ */
```

Comment:
It was mostly a test on that particular file to see how it works. It's easy to revert. Let's discuss that next week ;)